RELEASE.md

```diff
@@ -16,10 +16,12 @@ Please follow the established format:
 ## Bug fixes and other changes
 
 <!-- Add release notes for the upcoming release here -->
+- Remove Python 3.5 CI (#138)
```

Comment:
```suggestion
- Python 3.5 is no longer supported by the current and all future versions of Kedro-Viz (#138)
```

Reply:
Just updated this in the most recent commit. I try to use the present tense imperative (like in commit messages) in the release notes